SR2 Community Patch/scripts/gui/overlays/Quickbar.as

```diff
@@ -975,8 +975,7 @@ class LaborPlanets : ObjectMode {
 	}
 
 	bool filter(ObjectData@ dat) {
-		int minLabor = playerEmpire.hasTrait(getTraitID("Mechanoid")) ? 1 : 0;
-		if(dat.obj.getResourceProduction(TR_Labor) > minLabor)
+		if(dat.obj.getResourceProduction(TR_Labor) >= settings::dQuickbarMinLabor)
```

Comment:
This no longer checks if you're a Mechanoid. It should add +1 to the minimum if you're playing Mechanoids, IMO.

Reply:
(Fixing this was actually one of the tweaks I mentioned making. Still leaving this unresolved, in case anyone wants to discuss it.)